### PR TITLE
Hide host info from non-admin users

### LINKED ISSUE #933: Running VM has Host 'N/A' in Details for non-admin user
Running VM has Host N/A in Details for non-admin user. Admin user can see there name of the host where VM is running.

### PR DESCRIPTION
**Pull request by thaorell**
Fixes #933 

Work: Hide Host field from non-admin user in Details card on VM dashboard

### CODE REVIEW

File: src/components/VmDetails/cards/DetailsCard/index.js [hunk 1]
```diff
@@ -740,7 +741,7 @@ class DetailsCard extends React.Component {
                     <FieldRow label={msg.fqdn()} id={`${idPrefix}-fqdn`}>
                       { <EllipsisValue tooltip={fqdn}>{fqdn}</EllipsisValue> || <NotAvailable tooltip={msg.notAvailableUntilRunningAndGuestAgent()} id={`${idPrefix}-fqdn-not-available`} /> }
                     </FieldRow>
-                    <FieldRow label={msg.cluster()} id={`${idPrefix}-cluster`}>
+                    { isAdmin && <FieldRow label={msg.cluster()} id={`${idPrefix}-cluster`}>
```
Comment (sjd78) [inline src/components/VmDetails/cards/DetailsCard/index.js]: Logic is good, but I suggest a slight format change:
```jsx
  { isAdmin &&
    <FieldRow ...>
      ...
    </FieldRow>
  }
```

**Comment by isaranova** [comment]
ci build please

**Comment by isaranova** [comment]
The host and cluster fields are visible to the non-admin user, the host field is empty and cluster field has correct value. Admin user can see both fields and both contain correct values.

Screenshot_2019-09-05 VM Portal

**Comment by thaorell** [comment]
@isaranova  I've rebased and the cluster field seems to be doing well for non-admin user. For the host field, it's empty even on master. 
@sjd78 Do you know why Host's info might be empty? I believe it's a separate issue

**Comment by isaranova** [comment]
LGTM

**Pull request merged**